### ISSUE TITLE
Exception when running Craft update from CLI

### ISSUE THREAD

**Issue by juddlyon** [lindseydiloreto/craft-inventory#20]
Just upgraded to Craft Inventor v 3.0.0 on Craft 4.4.7.1. I could only upgrade with the plugin disabled due to the following error. It also appears when I enable the plugin after the update.

```php craft update all

Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getCookies()

Stack trace:
#0 /vendor/doublesecretagency/craft-inventory/src/Inventory.php(53): yii\base\Component->__call('getCookies', Array)
````

**Comment by lindseydiloreto** [comment]
Good news, this has [already been fixed and merged](https://github.com/doublesecretagency/craft-inventory/pull/19). (thanks @Jensderond!)

I'm making one more adjustment, and will be cutting a new release within the next hour or so.

If you can't wait that long, switch your Inventory version to this and run `composer update` again...

```js
"doublesecretagency/craft-inventory": "dev-v2-dev"
```

**Comment by juddlyon** [comment]
Thanks for the quick reply! It's all good, just running it disabled until the fix is ready.

**Comment by lindseydiloreto** [comment]
This patch is now available in [v3.0.1](https://github.com/doublesecretagency/craft-inventory/releases/tag/3.0.1) 👍
